### ISSUE TITLE
Shop

### ISSUE THREAD

**Issue by Clockshopla** [clockshop/clockshopwp#7]
Same Question - where to house the shop?

It seems to me that it has something to do with the options available in the engage section. I think we would rather change that setup and add it there.

**Comment by halfempty** [comment]
@Clockshopla I'm iffy on adding Shop to Engage. I fear it would cloud the message, there. What if... We nixed "Subscribe" from the Nav, and put Shop there? We could then add Subscribe to About. (It's also on Engage, already.)

**Comment by halfempty** [comment]
@Clockshopla 

> Do you have a suggestion on a good place to manage the sale of goods and items?

Yes. Gravity Forms, the plugin we use for the Volunteer Form, also works in conjunction with PayPal. I can walk you through its use. Do you have any free time to Skype today or tomorrow?

**Comment by Clockshopla** [comment]
@halfempty thanks for the suggestions. we have an event on thursday we were might sell out our current inventory, so we'd actually like to wait until early next week to revisit. the store question opened up other web questions to revisit later. thanks Marty

**Comment by Clockshopla** [comment]
@halfempty Hi so we ended up adding a temporary link until we are able to do something more extensive. Thank you!
